packages/core/src/util/jwt.ts

```diff
@@ -101,29 +110,31 @@ export async function verifyJwt(
   options = { ...defaultVerifyJwtOptions, ...options };
 
   const creds = getXsuaaServiceCredentials(token);
+  const jku = getVerificationKeyURL(token);
```

Comment:
Maybe `verificationKeyUrl` would be a better name for the variable.

Reply:
I already thought, ehh maybe a good name. And then Dennis and you comment on it so clear change.Done.